build/uuid-redirects.json

```
@@ -24,6 +24,13 @@
     "Compendium.pf2e.feat-effects.Item.KkbFlNfcQQUfSVXd": "Compendium.pf2e.feat-effects.Item.Effect: Sanctify Armament",
     "Compendium.pf2e.feat-effects.Item.XC3dRbwfu35vuvmM": "Compendium.pf2e.feat-effects.Item.Effect: Sanctify Armament",
     "Compendium.pf2e.pathfinder-bestiary-3.Actor.QUzBzxRy6HLeK7ja": "Compendium.pf2e.pathfinder-monster-core.Actor.Dybbuk",
+    "Compendium.pf2e.pathfinder-bestiary-2.Actor.fv91xoQJlogVbruW": "Compendium.pf2e.pathfinder-monster-core.Actor.Aapoph Serpentfolk",
+    "Compendium.pf2e.pathfinder-bestiary-2.Actor.WMiPblOVq5w2Fa1x": "Compendium.pf2e.pathfinder-monster-core.Actor.Aolaz",
+    "Compendium.pf2e.pathfinder-bestiary.Actor.cZDiyluplFqRxmGy": "Compendium.pf2e.pathfinder-monster-core.Actor.Animated Armor",
+    "Compendium.pf2e.pathfinder-bestiary.Actor.PiAGhPEzJMC2egQk": "Compendium.pf2e.pathfinder-monster-core.Actor.Animated Statue",
+    "Compendium.pf2e.pathfinder-bestiary.Actor.6CQEelygt968CB7m": "Compendium.pf2e.pathfinder-monster-core.Actor.Ankylosaurus",
+    "Compendium.pf2e.pathfinder-bestiary.Actor.c6AE2Mh8BRtBgbtz": "Compendium.pf2e.pathfinder-monster-core.Actor.Arboreal Regent",
+    "Compendium.pf2e.pathfinder-bestiary.Actor.aNWiP985fISjClGo": "Compendium.pf2e.pathfinder-monster-core.Actor.Arboreal Warden",
```

```suggestion
    "Compendium.pf2e.pathfinder-bestiary.Actor.cZDiyluplFqRxmGy": "Compendium.pf2e.pathfinder-monster-core.Actor.Animated Armor",
    "Compendium.pf2e.pathfinder-bestiary.Actor.PiAGhPEzJMC2egQk": "Compendium.pf2e.pathfinder-monster-core.Actor.Animated Statue",
    "Compendium.pf2e.pathfinder-bestiary.Actor.6CQEelygt968CB7m": "Compendium.pf2e.pathfinder-monster-core.Actor.Ankylosaurus",
    "Compendium.pf2e.pathfinder-bestiary.Actor.c6AE2Mh8BRtBgbtz": "Compendium.pf2e.pathfinder-monster-core.Actor.Arboreal Regent",
    "Compendium.pf2e.pathfinder-bestiary.Actor.aNWiP985fISjClGo": "Compendium.pf2e.pathfinder-monster-core.Actor.Arboreal Warden",
    "Compendium.pf2e.pathfinder-bestiary-2.Actor.fv91xoQJlogVbruW": "Compendium.pf2e.pathfinder-monster-core.Actor.Aapoph Serpentfolk",
    "Compendium.pf2e.pathfinder-bestiary-2.Actor.WMiPblOVq5w2Fa1x": "Compendium.pf2e.pathfinder-monster-core.Actor.Aolaz",
    "Compendium.pf2e.pathfinder-bestiary-3.Actor.QUzBzxRy6HLeK7ja": "Compendium.pf2e.pathfinder-monster-core.Actor.Dybbuk",
```
For ease of navigation, I've been using the left pack name to order things. Might as well be consistent.